docs/en/ingest-management/fleet/air-gapped.asciidoc

```diff
@@ -0,0 +1,126 @@
+[[air-gapped]]
+= Air-gapped environments
+
+There are certain environments in which network traffic restrictions are mandatory. In these environments, the {kib} instance
+isn't able to reach the public {package-registry} endpoints, like https://epr.elastic.co/[epr.elastic.co], to download
+package metadata and content.
+
+There are two workarounds in this situation -- use a proxy server as network gateway to reach the public endpoints,
+or deploy your own instance of the {package-registry}.
+
+[discrete]
+[[air-gapped-proxy-server]]
+== Use a proxy server
+
+If you can route traffic to the public endpoint of the {package-registry} through a network gateway, there is a property in {kib} that
+can orchestrate to use a proxy server:
+
+[source,yaml]
+----
+xpack.fleet.registryProxyUrl: your-nat-gateway.corp.net
+----
+
+[discrete]
+[[air-gapped-diy-epr]]
+== Host your own {package-registry}
+
+If routing traffic through a proxy server is not an option, you can host your own {package-registry}.
+
+The {package-registry} can be deployed and hosted on-site using one of the
+available Docker images. These docker images include the {package-registry} and
+a selection of packages.
+
+There are different distributions available:
+
+* {version} (recommended): +docker.elastic.co/package-registry/distribution:{version}+ - Selection of packages from the production repository released with the {version} version of the {stack}.
+* production: `docker.elastic.co/package-registry/distribution:production` - Packages available in the production registry (https://epr.elastic.co).
+* staging: `docker.elastic.co/package-registry/distribution:staging` - Packages available in the staging registry (https://epr-staging.elastic.co). These packages may be pending of validation.
+* snapshot: `docker.elastic.co/package-registry/distribution:snapshot` - Packages under development.
+
+ifeval::["{release-state}"=="unreleased"]
+[WARNING]
+====
+Version {version} of the {package-registry} distribution has not yet been released.
+====
+endif::[]
```

Comment:
@dedemorton this warning doesn't appear in the CI build https://observability-docs_1287.docs-preview.app.elstc.co/guide/en/fleet/master/air-gapped.html#air-gapped-diy-epr

Is this expected? or something is wrong here?

Reply:
Oh that's my fault! If you look at the index.asciidoc file, you'll see:

```
//TODO: Remove release-state override before merging
:release-state: released
```

This setting overrides the setting for the branch specified [here](https://github.com/elastic/docs/blob/master/shared/versions/stack/master.asciidoc).

The override is there so that reviewers can see all the content. You can remove the override in index.asciidoc or comment it out.